invatar/svg.py

```diff
@@ -28,14 +28,17 @@ def make_svg(**options):
 
     svg = """
     <svg xmlns="http://www.w3.org/2000/svg"
-         width="{size}px" height="{size}px"
-        style="background-color: {bg}">
-      <text y="50%" x="50%"
-          text-anchor="middle" dominant-baseline="central"
-          fill="{color}"
-          style="font-family: {font_family}; font-size: {font_size}px">
-        {text}
-      </text>
+         width="{size}px" height="{size}px">
+      <g>
+        <rect x="0" y="0" fill="{bg}" width="{size}px" height="{size}px">
+        </rect>
+        <text y="50%" x="50%" fill="{color}"
+              text-anchor="middle" dominant-baseline="central"
+              style="font-family: {font_family};
+              font-size: {font_size}px">
```

Comment:
Can you move `font-size` to the line above:
`style="font-family: {font_family}; font-size: {font_size}px"`